src/components/QR/qr.scss

```diff
@@ -0,0 +1,43 @@
+.qr-wrapper{
+    width: 100vw;
+    height: 100vh;
+    margin: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 2vh;
+    box-sizing: border-box;
+
+    .qr-header{
+        margin: 0;
+        padding: 0;
+    }
+}
+
+#qrimg {
+    margin: 0;
+    padding: 0;
+    width: 200px;
+    height: 200px;
+}
+
+@media screen and (min-width: 320px) {
```

Comment:
Let's just make a desktop version and a mobile version. No need to handle a bunch of different sized devices